Spigot-Server-Patches/0703-ANSI-support-for-server-GUI.patch

```diff
@@ -0,0 +1,272 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Callum Seabrook <yutubebroz@gmail.com>
+Date: Sun, 18 Apr 2021 15:17:58 +0100
+Subject: [PATCH] ANSI support for server GUI
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/gui/ANSITextPane.java b/src/main/java/com/destroystokyo/paper/gui/ANSITextPane.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..31405fdc15733c9ab88b2340a268552eac6c3d98
+--- /dev/null
++++ b/src/main/java/com/destroystokyo/paper/gui/ANSITextPane.java
+@@ -0,0 +1,120 @@
++package com.destroystokyo.paper.gui;
```

Comment:
Wrong package.

Reply:
I only put it in there since that's where all the other GUI stuff is. I'm assuming that I should relocate it to `io.papermc.paper.gui`?

Reply:
Yes